liquibase-standard/src/main/java/liquibase/database/Database.java

```diff
@@ -182,6 +196,103 @@ default String getDisplayName() {
+            } catch (LiquibaseException e) {
+                throw new UnexpectedLiquibaseException(e);
+            }
+
+            final long changeSetStarted = System.currentTimeMillis();
+            CompareControl compareControl = new CompareControl(
+                    new CompareControl.SchemaComparison[]{
+                            new CompareControl.SchemaComparison(
+                                    CatalogAndSchema.DEFAULT,
+                                    schemaToDrop)},
+                    snapshot.getSnapshotControl().getTypesToInclude());
+            DiffResult diffResult = DiffGeneratorFactory.getInstance().compare(
+                    new EmptyDatabaseSnapshot(this),
+                    snapshot,
+                    compareControl);
+
+            List<ChangeSet> changeSets = new DiffToChangeLog(diffResult, new DiffOutputControl(true, true, false, null).addIncludedSchema(schemaToDrop)).generateChangeSets();
+            Scope.getCurrentScope().getLog(getClass()).fine(String.format("ChangeSet to Remove Database Objects generated in %d ms.", System.currentTimeMillis() - changeSetStarted));
+
+            boolean previousAutoCommit = this.getAutoCommitMode();
+            this.commit(); //clear out currently executed statements
+            this.setAutoCommit(false); //some DDL doesn't work in autocommit mode
+            final boolean reEnableFK = supportsForeignKeyDisable() && disableForeignKeyChecks();
+            StringBuilder completeSql = new StringBuilder();
+            try {
+                for (ChangeSet changeSet : changeSets) {
+                    changeSet.setFailOnError(false);
+                    for (Change change : changeSet.getChanges()) {
+                        if (change instanceof DropTableChange) {
+                            ((DropTableChange) change).setCascadeConstraints(true);
+                        }
+                        SqlStatement[] sqlStatements = change.generateStatements(this);
+                        for (SqlStatement statement : sqlStatements) {
+                            AtomicReference<Sql[]> sqls = new AtomicReference<>(null);
+                            Scope.child(Collections.singletonMap(SqlGeneratorFactory.GENERATED_SQL_ARRAY_SCOPE_KEY, sqls), () -> {
+                                Scope.getCurrentScope().getSingleton(ExecutorService.class).getExecutor("jdbc", this).execute(statement);
+                            });
+                            if (StringUtil.isNotEmpty(completeSql.toString()) && !completeSql.toString().endsWith("; ")) {
+                                completeSql.append("; ");
+                            }
+                            completeSql.append(StringUtil.join(Arrays.stream(sqls.get()).map(Sql::toSql).collect(Collectors.toList()), "; "));
+                        }
```

Comment:
This is one notable change compared to the same method in `AbstractJdbcDatabase`